workers/scripts/run_mapping_saturation_models.py

```diff
@@ -0,0 +1,127 @@
+"""Run all statistical models of the Mapping Saturation indicator and plot them."""
+
+import asyncio
+import logging
+import pickle
+from os.path import exists
+
+import matplotlib.pyplot as plt
+import numpy as np
+from dacite import from_dict
+from rpy2.rinterface_lib.embedded import RRuntimeError
+
+import ohsome_quality_analyst.geodatabase.client as db_client
+import ohsome_quality_analyst.ohsome.client as ohsome_client
+from ohsome_quality_analyst.base.indicator import LayerDefinition
+from ohsome_quality_analyst.indicators.mapping_saturation import models
+from ohsome_quality_analyst.utils.definitions import get_layer_definition
+
+
+def plot(xdata, ydata, fitted_models):
+    plt.figure()
+    prev_ax = None
+    for i, model in enumerate(fitted_models):
+        if i == 0:
+            ax = plt.subplot(2, 3, i + 1)
+        else:
+            ax = plt.subplot(2, 3, i + 1, sharex=prev_ax, sharey=prev_ax)
+        ax.plot(xdata, ydata, label="OSM Data")
+        ax.plot(
+            xdata,
+            model.fitted_values,
+            label="Mean Absolute Error: {0}".format(model.mae),
+        )
+        ax.axhline(y=model.asymptote, color="pink", linestyle="--", label="Asymptote")
+        ax.set_title(model.name)
+        ax.legend()
+        ax.set(xlabel="time", ylabel="features")
+        ax.label_outer()
+        prev_ax = ax
+    plt.show()
+
+
+async def query_ohsome_api(features, layers) -> list:
```

Comment:
Avoid shadowing variables from outer scope.

Reply:
This should be fixed now